web/themes/custom/cloudy/templates/field/field.html.twig

```diff
@@ -65,24 +65,30 @@
   ])
 %}
 
-{% spaceless %}
+{% apply spaceless %}
   {% block field %}
   <div{{ attributes.addClass(field_classes).addClass(classes) }}>
     {% block label %}
       {% if label_display == "above" %}
         <h2{{ title_attributes.addClass(title_classes) }}>{{ label }}</h2>
-      {% elseif not label_hidden %}
+      {% elseif label_display != "hidden" %}
```

Comment:
I'm curious why you changed this line.

Reply:
I think I swapped it because the statement wasn't triggering for me with label_hidden. It should have, but both do the same logic.